### PR TITLE
Manage domain change

### PR DESCRIPTION
**Pull request by AlexBegin**

### CODE REVIEW

File: examples/ReactJsSdkApp/README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ python3 client_backend.py -sk <private_key> -pk <pti_public_key> -c <client id>
 * `private_key`: path to a JWS json file containing your private key. It must match the public key in the endpoint you are targeting on the PTI platform
 * `pti_public_key`: path to PTI's prod public key. That key should be used to verify the signatures of webhook calls made by PTT. This public key is the `utils/pti-prod-public.jwk` provided in this repository.
 * `client_id`: ID provided to you during onboarding. This ID must have been setup by PTI in the targeted environment.
-* `pti_api_url`: Root url of the targeted PTI API. E.g. `https://pti.apistaging.pticlient.com/v0`
+* `pti_api_url`: Root url of the targeted PTI API. E.g. `https://api.fiant.io/v1`
```
Comment (remicartierPTI) [inline examples/ReactJsSdkApp/README.md:41]: staging vs prod, ok ?

File: guide/v0/kyc.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ this would lead to a KYC check with a KYC tier of 2. A transaction request that
 
 ```js
 const callTransactionLog = (accessToken) => {
-        const baseUrl = 'https://pti.apistaging.pticlient.com/v0';
+        const baseUrl = 'https://api.fiant.io/v0';
```
Comment (remicartierPTI) [inline guide/v0/kyc.md:95]: staging vs prod, ok ?

File: guide/v1/examples.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ Deposit transactions are used to bring in assets in the client ecosystem.
 
 ```js
 const callAssessment = (accessToken) => {
-        const baseUrl = 'https://apistaging.pticlient.com/v1';
+        const baseUrl = 'https://api.fiant.io/v1';
```
Comment (remicartierPTI) [inline guide/v1/examples.md:36]: staging vs prod, ok ?

File: guide/v1/examples.md [hunk 4]
```diff
@@ -83,7 +83,7 @@ const callAssessment = (accessToken) => {
 Swaps are used to exchange assets within the ecosystem.
 ```js
 const callAssessment = (accessToken) => {
-        const baseUrl = 'https://apistaging.pticlient.com/v1';
+        const baseUrl = 'https://api.fiant.io/v1';
```
Comment (remicartierPTI) [inline guide/v1/examples.md:86]: staging vs prod, ok ?

File: guide/v1/examples.md [hunk 5]
```diff
@@ -142,7 +142,7 @@ Withdrawals are used to take assets out of the ecosystem.
 
 ```js
     const callTransactionLog = (accessToken) => {
-        const baseUrl = 'https://apistaging.pticlient.com/v1';
+        const baseUrl = 'https://api.fiant.io/v1';
```
Comment (remicartierPTI) [inline guide/v1/examples.md:145]: staging vs prod, ok ?

File: guide/v1/kyc.md [hunk 6]
```diff
@@ -92,7 +92,7 @@ this would lead to a KYC check with a KYC tier of 2. A transaction request that
 
 ```js
 const callTransactionLog = (accessToken) => {
-        const baseUrl = 'https://pti.apistaging.pticlient.com/v1';
+        const baseUrl = 'https://api.staging.fiant.io/v1';
```
Comment (remicartierPTI) [inline guide/v1/kyc.md:95]: staging vs prod, ok ?

File: examples/ReactJsSdkApp/README.md (hunk 1 above)
Comment (AlexBegin) [reply examples/ReactJsSdkApp/README.md:41]: I mean, this is just doc, dont know why it would be staging or prod or dev

**Review by remicartierPTI — APPROVED** [review]

**Pull request merged**